src/renderers/dom/ReactDOM.js

```diff
@@ -72,6 +72,7 @@ if (__DEV__) {
   var ExecutionEnvironment = require('ExecutionEnvironment');
   if (ExecutionEnvironment.canUseDOM && window.top === window.self) {
 
+    var isFileUrl = location.protocol.indexOf('http') === -1;
```

Comment:
Nit: We could probably move this line into the two `if` statements below, immediately above the `console.debug`.  Keeping it closer to the console.debug makes the code easier to read, since that's the only place it's used.

(also makes the code a microsecond faster, in the case where we don't end up printing the message, not that it's a huge difference :P; readability is the primary factor here)